website/projects/map4sci/src/app/map/map.module.ts

```diff
@@ -0,0 +1,18 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { NgxMapboxGLModule } from 'ngx-mapbox-gl';
+
+import { MapComponent } from './map.component';
+
+
+@NgModule({
+  imports: [
+    CommonModule,
+    NgxMapboxGLModule.withConfig({
+      accessToken: 'pk.eyJ1IjoiZWRsdTc3NyIsImEiOiJja2wycDV2dzAwazg1MnFxamd2dDFmdmlyIn0.UxjI5qHgJIN5NZFJRi37DA',
```

Comment:
Please try without an access token now. I know for a fact that we didn't need it for at least two other projects. See [this search](https://github.com/search?q=org%3Acns-iu+NgxMapboxGLModule&type=code)